bool need_frontier
---
geometry_msgs/PoseStamped goal